lib/msf/core/transport_config.rb

```diff
@@ -0,0 +1,71 @@
+# -*- coding: binary -*-
+
+require 'msf/core/payload/uuid_options'
+
+##
+# This module contains helper functions for creating the transport
+# configuration stubs that are used for Meterpreter payloads.
+##
+module Msf::TransportConfig
```

Comment:
Is there a reason this isn't under `Msf::Payload`? It should probably be `lib/msf/core/payload/transport_config.rb`